breeze-complete

```diff
@@ -138,7 +138,7 @@ _breeze_long_options="
 help python: backend: integration:
 kubernetes-mode: kubernetes-version: helm-version: kind-version:
 skip-mounting-local-sources install-airflow-version: install-airflow-reference: db-reset
-verbose assume-yes assume-no assume-quit forward-credentials start-airflow
+verbose assume-yes assume-no assume-quit forward-credentials start-airflow init-script:
```

Comment:
is the trailing ":" (colon) here intended?

Reply:
I assumed the colon was for flags accepting a parameter, --init-script needs the name of the file. Did I got it wrong?

Reply:
Oh yes, you got it right